src/main/io/ledstrip.c

```diff
@@ -131,6 +131,7 @@ void pgResetFn_ledStripConfig(ledStripConfig_t *ledStripConfig)
     ledStripConfig->ledstrip_beacon_percent = 50;       // 50% duty cycle
     ledStripConfig->ledstrip_beacon_armed_only = false; // blink always
     ledStripConfig->ledstrip_visual_beeper_color = VISUAL_BEEPER_COLOR;
+    ledStripConfig->ledstrip_brightness = 100;
```

Comment:
Changing the parameter group also needs the PG version to be changed from version `1` to version `2`. 

So you got to change `line 118` to
```cpp
PG_REGISTER_WITH_RESET_FN(ledStripConfig_t, ledStripConfig, PG_LED_STRIP_CONFIG, 2);
```

Reply:
> Changing the parameter group also needs the PG version to be changed from version `1` to version `2`.

This is incorrect in this case. See: https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Thank you for that! I’ll read the doc and amend accordingly.

Reply:
Actually @KarateBrot isn't entirely right - in general, if the change to the parameter group is only appending of new members at the end then a version increment (causing a reset to defaults) is not needed, as missing elements at the end will simply be initialised to 0. HOWEVER if the appended elements have defaults different to 0 (like in this case), a version increment is needed to get them initialised to the correct value by the reset.

See 
https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md for all of the gory details.